web-common/src/runtime-client/invalidation.ts

```diff
@@ -60,6 +60,7 @@ export async function invalidateAllMetricsViews(
 ) {
   // First, refetch the resource entries (which returns the available dimensions and measures)
   await queryClient.refetchQueries({
+    type: "active",
```

Comment:
Does this not lead to stale data in dashboards not in view?

Reply:
Yes and no. The only time this function is called (directly or not) is in the context of updating the user. Viewing as a user is specific to a dashboard, not a project. So, yes, only the currently active queries for the dashboard you're looking at are refetched. The data of other dashboards is never refetched/invalidated/etc, so it's not so much "stale" as it is unaltered.